File: Angular/App/AngularApp/src/_model/commentDetail.ts
import { User } from './user';

export class CommentDetail {
  id: number;
  postDetaild_Id: number;
  comment: string;
  likeCount: number;
  dislikeCount: string;
  active: boolean;
  author: number;
  editor: number;
  modified: Date;
  isLiked: boolean;
  isDisLike:boolean;
  createdP: Date;
  user: User;
  editUser: User;
  authorUser: User;
}

File: Model/CommentDetail.cs
namespace Model
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("CommentDetail")]
    public partial class CommentDetail
    {
        public long Id { get; set; }

        public long PostDetail_Id { get; set; }

        public string Comment { get; set; }

        public long LikeCount { get; set; }

        public long? DislikeCount { get; set; }

        public long? Author { get; set; }
        public long? Editor { get; set; }

        public DateTime? Created { get; set; }

        public DateTime? Modified { get; set; }

        public bool Active { get; set; }

        public virtual PostDetail PostDetail { get; set; }
    }
}
